benchmarks/oss.py

```diff
@@ -109,7 +109,8 @@ def validate_benchmark(measurements, final_loss, args, check_regression):
         assert max_memory < 1.05 * golden_data["reference_memory"], (
             f"Memory use regression detected: " f"{max_memory} vs. {1.05* golden_data['reference_memory']}"
         )
-        assert abs(cast(float, final_loss) - golden_data["reference_loss"]) < 1e-2, (
+        # any min_loss < than golden + epsilon is OK.
```

Comment:
I know it's being discussed, but basically OSS is in pytorch already and FSDP is a superset of shardedddp feature-wise, so this could also be removed altogether to cut on the maintenance costs

Reply:
Agreed. I think we will have a cleanup change separately. I am in favor of retiring them soon.